src/test/java/org/opentripplanner/TestServerContext.java

```diff
@@ -49,6 +51,7 @@ public static OtpServerRequestContext createServerContext(
       createEmissionsService(),
       routerConfig.flexConfig(),
       List.of(),
+      new DefaultStopConsolidationService(new DefaultStopConsolidationRepository(), transitModel),
```

Comment:
Consider using the same factory method pattern that the above code use. The EmissionsService and the StopConsolidationService should not be injected here since they are Sandbox services. This enables these services under test and allow them to run code.

Reply:
I can actually just use `null`.

Adressed in f671f54e92444fa8533e5669f36ecd13ef00204e.